platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -675,6 +675,16 @@ This behavior previously caused a warning: `Gradle Module Metadata is modified a
 
 It will now fail with an error, suggesting a review of the relevant documentation.
 
+==== New subtypes of ComponentIdentifier introduced
```

Comment:
Can you move this to `=== Updated Defaults and APIs`? It's in the wrong section IMHO.